src/openrct2/drawing/Drawing.h

```diff
@@ -195,10 +195,12 @@ enum class FilterPaletteID : int32_t
     PaletteLandMarker,
     Palette34,
 
-    PaletteWaterMarker = Palette34 + 4,
+    Palette37,
```

Comment:
This should be `Palette37 = Palette34 + 3,`

Reply:
I remove while test something and forgot to add back, fix now.